ara/plugins/callback/ara_default.py

```diff
@@ -191,6 +191,16 @@
         key: callback_threads
 """
 
+# Task modules for which ara should save host facts
+ANSIBLE_SETUP_MODULES = [
+    "setup",
+    "ansible.builtin.setup",
+    "ansible.legacy.setup",
+    "gather_facts",
+    "ansible.builtin.gather_facts",
+    "ansible.legacy.setup",
+]
```

Comment:
```suggestion
ANSIBLE_SETUP_MODULES = frozenset([
    "setup",
    "ansible.builtin.setup",
    "ansible.legacy.setup",
    "gather_facts",
    "ansible.builtin.gather_facts",
    "ansible.legacy.setup",
])
```

Reply:
haha 😂 Now you are just comment to comment anything, aren't you?

Reply:
I don't have strong feelings either way -- this particular list is meant to be a constant (you know, by convention, top of the file, capitalized variable) but I guess it's mutable unless it's in a frozenset.

I wouldn't have thought of it myself and so it wasn't in my suggestion, I don't use it very often :)

Feel free to commit the suggestion, I'll squash it before merging.